config/default.ini

```diff
@@ -0,0 +1,23 @@
+[KEYS]
+YouTube=
+DiscordToken=
+TwitchClientID=
+
```

Comment:
Include useful comments on where to find these keys. Perhaps also create rules in the configparser that automatically switch off commands that rely on the keys if the aren't set.